src/leap/bitmask/gui/mail_status.py

```diff
@@ -233,12 +233,15 @@ def _set_mail_status(self, status, ready=0):
         self._action_mail_status.setText(tray_status)
         self._update_systray_tooltip()
 
-    def _mail_handle_soledad_events(self, event, content):
+    def _mail_handle_soledad_events(self, event, uuid, content=""):
```

Comment:
Ok, I think I see where this is comming, I still don't understand why `uuid`. I think it's actually a dict `{'uuid': str, 'userid': str}`, isn't it?